lib/utils/validate-email.ts

```diff
@@ -1,7 +1,4 @@
-export const validateEmail = (email: string) => {
-  return String(email)
-    .toLowerCase()
-    .match(
-      /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/
-    );
-};
+const emailRegexp =
+  /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-z\-0-9]+\.)+[a-z]{2,}))$/i;
```

Comment:
Which regex is this? On the backend API, we use the HTML 5 email validator from the web standard spec:

https://github.com/open-sauced/api/blob/9996046f4c21789c40d54d30d9b6b5a4ed4d02cb/src/common/util/email.ts#L3-L17

It looks like this drifts abit. And I'd prefer if we aligned these.

Reply:
Hi, this is the same one as before, I just extracted it to prevent it from being rebuilt . Will update it to the one from the backend.

Reply:
Found the failure issue as well, the new regexp allows top level domains for emails, so adjusted the test to match that